release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -450,6 +450,46 @@ For more information, see the following sections:
 [id="ocp-4-15-storage"]
 === Storage
 
+[id="ocp-4.15-storage-recovering-vgs-from-prev-installation"]
+==== Recovering volume groups from the previous {lvms} installation
+With this release, `LVMCluster` custom resource (CR) provides support for recovering volume groups from the previous {lvms} installation. If the `deviceClasses.name` field is set to the name of a volume group from the previous {lvms} installation, {lvms} recreates the resources related to that volume group in the current {lvms} installation. This simplifies the process of using devices from the previous {lvms} installation through the re-installation of {lvms}.
+
+For more information, see xref:..//storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-creating-lvms-cluster_logical-volume-manager-storage[Creating an `LVMCluster` custom resource (CR)].
```

Comment:
All of your xrefs do resolve, but they each have an extra `/` at the front.

Reply:
Removed the extra "/".